### PR TITLE
Migrate to gulp4 and update stackblitz to handle multiple demos base urls

### LINKED ISSUE #155: Configure Stackblitz handler to handle multiple different demo ulrs
Currently the stackblitz handler operates with the demo base url  only of the first sample. Since we have now multiple different demo urls, it must be configured to handle all of them

### PR DESCRIPTION
**Pull request by HristoP96**
Closes #155

### CODE REVIEW

**Comment by HristoP96** [comment]
In order this PR to work, you have to change the demosBaseUrl or lobDemosBaseUrl in the preconfig.json with the respective url on which the angular samples are served

File: gulpfile.js [hunk 1]
```diff
@@ -63,29 +63,34 @@ gulp.task("bundle-and-minify", () => {
...
+        var hash = md5File.sync(filePath);
+        if (md5HashMap[filePath] !== hash) {
+            md5HashMap[filePath] = hash;
         }
+ 
     });
-});
+    done();
+}
+// gulp.task("bundle-and-minify:watch", ["bundle-and-minify", "generate-file-check-sum-map"], () => {
```
Comment (Aleksandyr) [inline gulpfile.js]: remove commented rows

File: gulpfile.js [hunk 2]
```diff
@@ -96,9 +101,12 @@ gulp.task("generate-file-check-sum-map", () => {
         var hash = md5File.sync(filePath);
         md5HashMap[filePath] = hash;
     });
-})
+    done();
+}
+
+// gulp.task("generate-file-check-sum-map", )
```
Comment (Aleksandyr) [inline gulpfile.js]: remove commented rows

File: gulpfile.js [hunk 3]
```diff
@@ -63,29 +63,34 @@ gulp.task("bundle-and-minify", () => {
...
     });
 
-    return Promise.all(promises).then(function () {
-        gulp.start("generate-bundling-global-metadata");
-    });
-});
+    return Promise.all(promises).then(generateBundlingGlobalMetadata(done));
+}
+
+// gulp.task("bundle-and-minify", );
```
Comment (Aleksandyr) [inline gulpfile.js]: remove commented rows

**Comment by HristoP96** [comment]
@Aleksandyr  the comments are removed

**Pull request merged**